src/github-actions/actions.ts

```diff
@@ -0,0 +1,56 @@
+import { ActionMetadata, RunsUsing } from "../github/actions-metadata-model";
+import { TypeScriptProject, TypeScriptProjectOptions } from "../typescript";
+import { YamlFile } from "../yaml";
+import { GitHubActionMetadata } from "./github-action-metadata";
+
+/**
+ * Properties for creating a GitHubActionTypeScriptProject
+ */
+export interface GitHubActionTypeScriptOptions
+  extends TypeScriptProjectOptions {
+  /**
+   * Every GitHub Action must have a metadata file named `action.yml`.
+   * Projen will manage this file for you using the specifications of
+   * this property.
+   */
```

Comment:
Trying to tweak the API description to be a bit more intent based than implementation based

```suggestion
  /**
   * Options for the GitHub Action metadata stored in `action.yml`
   *
   * @default - an action named after the project `name` that runs from dist/index.js
   */
```